products/memgraph/upcoming/tutorials/football-transfers.md

```diff
@@ -0,0 +1,274 @@
+## Football transfers
+
+This article is a part of a series intended to show how to use Memgraph
+on real-world data to retrieve some interesting and useful
+information.
+
+### Introduction
+
+Football is a word that could mean one of several sports.In this article,
+we are referring to the best-known type of football, association football.
+In North America, South Africa, and Australia to avoid confusion with other
+types of football, this is called "soccer". It is a team sport played with
+a spherical ball between two teams of 11 players.  But more often than not,
+there are more than just 11 players in every team. 
+
+There are a few things to consider in this article. First of all,
+the transfers. 
+
+In professional football, a transfer is the action taken whenever a player
+under contract moves between teams. It refers to the transferring of
+a player's registration from one association football club to another.
+In general, the players can only be transferred during a transfer window
+and according to the rules. The Transfer window is a period during the year
+in which a football team can transfer players. There are usually two transfer windows
+in the same season, one in winter that lasts through January,
+and one in summer that lasts through July and August.
+
+Usually some sort of compensation is paid for the player's rights,
+which is known as a transfer fee. When a player moves from one team to another,
+their old contract is terminated and they negotiate a new one with the team
+they are moving to. In some cases, however, transfers can function similarly
+to player trades, as teams can offer another player on their team as part of the fee.
+
+Of course you may presume, there is a lot of money in the game.
```

Comment:
Wording: `As you may presume, ...`, `...in the game of transfers.`

Reply:
:+1: